client/lib/pages/main_pages/routes.dart

```diff
@@ -35,9 +37,12 @@ class Routes {
           dataSource: Provider.of<ContentStore>(context),
           title: S.of(context).homePagePageButtonQuestions,
         ),
-    '/symptom-checker': (context) => CheckUpPosterPage(
+    '/symptom-checker-survey': (context) => SymptomCheckerView(
```

Comment:
Does this need to preserve the `/symptom_checker` route so that it can be toggled in to use?